src/nft/utils/asset.tsx

```diff
@@ -25,8 +40,43 @@ export const getAssetHref = (asset: GenieAsset | WalletAsset, origin?: DetailsOr
   return `/nfts/asset/${address}/${asset.tokenId}${origin ? `?origin=${origin}` : ''}`
 }
 
-export const getMarketplaceIcon = (marketplace: string) => {
-  return `/nft/svgs/marketplaces/${marketplace.toLowerCase()}.svg`
+export const getMarketplaceIcon = (
+  marketplace: string,
+  width: string | number,
+  height: string | number,
+  gradientId?: string
```

Comment:
if you're passing in uuidv4 here every time, do you need to pass it in? would there be a case you don't use it for the gradientId?